test_model.py

```diff
@@ -173,6 +176,9 @@ def compute_true_positive_rate(forecasted_anomalies, labeled_anomalies):
         # log some accuracy metrics here
         MLFLOW_CLIENT.log_metric(mlflow_run_id, "RMSE", rmse, metric_timestamp, item)
         MLFLOW_CLIENT.log_metric(mlflow_run_id, "Accuracy", accuracy, metric_timestamp, item)
+        MLFLOW_CLIENT.log_metric(mlflow_run_id, "Ground truth anomalies", ground_truth_anomalies, metric_timestamp, item)
+        MLFLOW_CLIENT.log_metric(mlflow_run_id, "Forecasted anomalies", predicted_anomalies, metric_timestamp, item)
+        MLFLOW_CLIENT.log_metric(mlflow_run_id, "Number of test data points", len(true_values.metric_values))
```

Comment:
Can you pass the `metric_timestamp` and item parameters to the `log_metric` function?